go-controller/pkg/ovn/multipolicy_test.go

```diff
@@ -540,9 +541,10 @@ var _ = ginkgo.Describe("OVN MultiNetworkPolicy Operations", func() {
 		})
 
 		ginkgo.DescribeTable("correctly adds and deletes pod IPs from secondary network namespace address set",
-			func(topology string, remote bool) {
+			func(topology string, remote bool, expectMissingMappingFailure bool) {
 				app.Action = func(ctx *cli.Context) error {
 					var err error
+					os.Setenv("K8S_NODE", "node1")
```

Comment:
shouldn't we clear this also on defer cleanup ?...

I worry this'll be a side effect that might impact other tests running in parallel.